packages/actions/src/Exports/Jobs/ExportCompletion.php

```diff
@@ -74,6 +74,14 @@ public function handle(): void
                     $this->formats,
                 )),
             )
-            ->sendToDatabase($this->export->user, isEventDispatched: true);
+            ->when(
+                $this->connection === 'sync',
+                fn (Notification $notification) => $notification->persistent()
+            )
+            ->when(
+                $this->connection === 'sync',
+                fn (Notification $notification) => $notification->send(),
+                fn (Notification $notification) => $notification->sendToDatabase($this->export->user, isEventDispatched: true),
+            );
```

Comment:
Maybe merge these?

```suggestion
            ->when(
                $this->connection === 'sync',
                fn (Notification $notification) => $notification
                    ->persistent()
                    ->send(),
                fn (Notification $notification) => $notification->sendToDatabase($this->export->user, isEventDispatched: true),
            );
```

Reply:
Thanks, for sure committed! Reason I kept it separate was that the other identifiers above also were in it's own `->when()`'s each, but this is also perfect.

Reply:
Yeah no big deal, just like grouping the conditions a little better.